CONTRIBUTING.md

```diff
@@ -30,7 +30,7 @@ These are just guidelines, not rules, use your best judgment and feel free to pr
 
 ### Code of Conduct
 
-This project adheres to the [Contributor Covenant 1.2](http://contributor-covenant.org/version/1/2/0).
+This project adheres to the [Contributor Covenant 1.3](http://contributor-covenant.org/version/1/3/0)  (see [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md)).
```

Comment:
@zeke I think @lee-dohm wanted to say that this should only link to the CODE_OF_CONDUCT.md file, not to http://contributor-covenant.org/version/1/3/0. The CODE_OF_CONDUCT.md file contains a link to http://contributor-covenant.org/version/1/3/0 (notice the section at the bottom), so it's not needed here and it might be confusing for users who will click that link first, land on http://contributor-covenant.org/version/1/3/0, and not find an email address filled out there. 

I think we should link to CODE_OF_CONDUCT.md only, both here and in the README, and link to http://contributor-covenant.org from CODE_OF_CONDUCT.md (as it is now).